sdk/src/main/java/org/torusresearch/customauth/types/AggregateVerifierParams.java

```diff
@@ -1,19 +1,21 @@
 package org.torusresearch.customauth.types;
 
+import org.torusresearch.torusutils.types.VerifyParams;
+
 public class AggregateVerifierParams {
```

Comment:
We should rather construct this object completely in the constructor vs setting individual items.

Reply:
This still needs to be resolved.

Reply:
resolved in #31